angular-rails-app/db/migrate/20170225185334_init_schema.rb

```diff
@@ -129,10 +129,12 @@ def up
       t.string   "borough",             limit: 255
       t.integer  "block",               limit: 4
       t.integer  "lot",                 limit: 4
+      t.integer  "bbl",                  limit: 8
       t.integer  "hpd_registration_id", limit: 4
     end
 
     add_index "properties", ["borough", "block", "lot"], name: "index_r_properties_on_borough_and_block_and_lot", unique: true, using: :btree
+    add_index "properties", ["bbl"], name: "index_r_properties_on_bbl", using: :btree
```

Comment:
I did make it unique, originally, but it seemed like it janked up the import? However, just tried it again and it seemed to work. Will add.